### PR TITLE
Add Welcome Guide to Help menu

### LINKED ISSUE #32: Hard to find the welcome page
I just downloaded Atom, and the welcome page was beautiful.  I sure wanted to read it.  But I closed Atom so I could move it from "downloads" to "desktop", and now I can't make the welcome page come back!

Searching for "welcome" in the help menu search bar gives zero results.

I also could not find a shortcut to re-open the welcome page by searching online.

Please consider adding a link to the welcome page to the "file", "view", or "help" menus.

### PR DESCRIPTION
**Pull request by mnquintana**
Resolves #32

### CODE REVIEW

File: menus/welcome.cson [hunk 1]
```diff
@@ -0,0 +1,6 @@
+'menu': [
+  'label': 'Help'
+  'submenu': [
+    { 'label': 'Welcome Guide', 'command': 'welcome:show'}
```
Comment (lee-dohm) [inline menus/welcome.cson]: *Nitpicky:* Space before closing brace?

**Comment by lee-dohm** [comment]
Other than the one little nit, looks fine to me!

File: menus/welcome.cson (hunk 1 above)
Comment (mnquintana) [inline menus/welcome.cson]: Well that's embarrassing

**Comment by maxbrunsfeld** [comment]
:ship:

**Comment by benogle** [comment]
Thanks for this

**Pull request merged**